packages/insomnia-app/app/main/window-utils.ts

```diff
@@ -20,11 +20,20 @@ import { clickLink, getDataDirectory, restartApp } from '../common/electron-help
 import * as log from '../common/log';
 import LocalStorage from './local-storage';
 
-const { app, Menu, shell, dialog, clipboard } = electron;
+const { app, Menu, shell, dialog, clipboard, session } = electron;
 // So we can use native modules in renderer
 // NOTE: This was (deprecated in Electron 10)[https://github.com/electron/electron/issues/18397] and (removed in Electron 14)[https://github.com/electron/electron/pull/26874]
 app.allowRendererProcessReuse = false;
 
+app.on('ready', () => {
+  // There's no option that prevents Electron from fetching spellcheck dictionaries from
+  // Chromium's CDN and passing a non-resolving URL is the only known way to prevent it from
+  // fetching.  https://github.com/electron/electron/issues/22995
```

Comment:
nit: no reason to break lines unnecessarily:
![Screenshot_20210914_202710](https://user-images.githubusercontent.com/15232461/133351120-e97e750f-4af5-4b2d-ae4f-1488d7123bf5.png)

```suggestion
  // There's no option that prevents Electron from fetching spellcheck dictionaries from Chromium's CDN and passing a non-resolving URL is the only known way to prevent it from fetching.
  // see: https://github.com/electron/electron/issues/22995
```